src/components/Button.js

```diff
@@ -1,14 +1,18 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
 class Button extends React.Component {
   render() {
-    return <button>{this.props.name}</button>;
+    const wide = this.props.wide ? 'fifty-percent' : '';
+    const color = this.props.color || '';
```

Comment:
Hey, if there is no prop sent, the color should be set from default props, we had that in the last milestone, what happened?
- you have css set for some button already let's not do that. use default props.

Reply:
Thanks so much for pointing that out. Now amended.